### ISSUE TITLE
Define a new command line executed always first (Just before the command of -c option)

### ISSUE THREAD

**Issue by greymd** [greymd/tmux-xpanes#85]
### Idea

When the multiple -c options are given like this

```
$ xpanes -c 'echo 1@{}' -c 'echo 2@{}' hoge 
```

All the arguments of -c options are combined with new lines and result is like this.

```
$ echo 1@hoge
1@hoge
$ echo 2@hoge
2@hoge
```

### Why this is helpful?

I think some users hope to change the default behavior.
The change what the user hopes can be implemented with use's customize command line like #84 .
If the multiple `-c` options are allowed, users can define alias of `xpanes` very easily, like this.

```
alias xpanes="xpanes -d -c original_command_always_first"
```

Creating alias of `xpanes` command is helpful idea (like #64).
And this feature allows users customize tmux-xpanes's behavior more easily and flexibly.

**Comment by greymd** [comment]
I noticed that "Multiple -c options" does not suit to Pipe mode.

```bash
# This is not allowed
$ echo AAA BBB | xpanes commandA -c commandB -c commandC...
```

My another idea is, implementing `-B` (Begin) and `-E`(End) options.
This naming rule is inspired by by php-cli.

```bash
$ xpanes -B 'ls -al' -c 'echo {}' -E 'clear' AAA BBB CCC

#### Following commands would be executed.
### 1) ls -al
### 2) echo AAA
### 3) clear

# Pipe mode (same result)
$ echo AAA BBB CCC | xpanes -B 'ls -al' -E 'clear' echo
```

**Comment by greymd** [comment]
I think it is helpful idea.
Close.
open #99 instead.

**Comment by greymd** [comment]
`-B` option will be released because I found some use cases which requires `-B` option.
Let me re-open it.
I an not sure `-E` option is worth implementing.

**Comment by greymd** [comment]
Released at [v4.1.0](https://github.com/greymd/tmux-xpanes/releases/tag/v4.1.0)
